akka-stream-tests/src/test/scala/akka/stream/impl/fusing/GraphInterpreterPortsSpec.scala

```diff
@@ -384,6 +384,8 @@ class GraphInterpreterPortsSpec extends AkkaSpec with GraphInterpreterSpecKit {
     "ignore pull while completing" in new PortTestSetup {
       out.complete()
       in.pull()
+      // While the pull event is not enqueued at this point, we should still report the state correctly
+      in.hasBeenPulled should be(true)
```

Comment:
@rkuhn and I have thought that no more logic bugs are present in the interpreter ;)